recipes/x86_64_level/install_scripts.sh

```diff
@@ -0,0 +1,10 @@
+mkdir -p "${PREFIX}"/etc/conda/{de,}activate.d/
+
+if [[ "$level" == "1" ]]; then
+  flag="-march=x86-64"
+else
+  flag="-march=x86-64-v{{ level }}"
+fi
+
+echo 'export CXXFLAGS="${CXXFLAGS} '$flag'"' >> "${PREFIX}"/etc/conda/activate.d/~activate-x86-64-level.sh
```

Comment:
Do we need to remove any other -march flags or do they add in the compiler?

Reply:
Also what about CFLAGS?

Reply:
> Do we need to remove any other -march flags or do they add in the compiler?

Compiler selects the last one.